pom.xml

```diff
@@ -130,7 +130,7 @@
     <snappy.version>1.1.1.7</snappy.version>
     <shiro.version>1.2.1</shiro.version>
     <slf4j.version>1.7.5</slf4j.version>
-    <tephra.version>0.6.3</tephra.version>
+    <tephra.version>0.6.5-SNAPSHOT</tephra.version>
```

Comment:
We were planning on upgrading to tephra-0.6.4 in cdap-3.4, but due to TEPHRA-169 we needed a new Tephra release, and hence the skip.